### PR TITLE
Handle multiple transaction pairs

### PR DESCRIPTION
**Pull request by honzajavorek**
Skipping empty requests causes bad request-response pairing in the Apiary Mock Server. There are probably only legacy reasons why the empty requests are getting skipped and with API Elements we should update.

I assumed the versioning scheme from reading 9c6fc1b890dfa3d4a7e362efebcaea6d2f0d1128.

**BREAKING CHANGE**

### CODE REVIEW

**Comment by honzajavorek** [comment]
I think this should affect only docs and mock server. I'll verify manually the tested blueprint renders correct docs as expected with this change in metamorphoses.

**Comment by honzajavorek** [comment]
To my surprise, the docs render correctly with this change.

File: package.json [hunk 1]
```diff
@@ -40,9 +40,10 @@
     "coffeeify": "^1.1.0",
     "coffeelint": "^1.11.1",
     "coveralls": "^2.11.0",
+    "drafter": "^1.2.0",
     "istanbul": "^0.4.5",
     "mocha": "^2.3.0",
-    "protagonist": "^1.6.4",
+    "protagonist": "^1.6.8",
```
Comment (kylef) [inline package.json:49]: Maybe this can be removed completely in favour of the introduced drafter?

Comment (honzajavorek) [reply package.json:49]: I think it's a good idea, but I don't want to snowball this PR much. I'll take a look and if the change is straightforward, I'll do it.

Comment (honzajavorek) [reply package.json:49]: Hmm, there's enough `protagonist.parse(..., {type: 'ast'}, ...` in the codebase for me not to interfere 😬

**Review by JackuB — APPROVED** [review]
Suspicious code, but can't find any more info on that. Looks like it shouldn't be there

**Comment by honzajavorek** [comment]
@JackuB thx! if anything, we'll hopefully learn whether something blows up when I'll make a PR in the core app

File: test/multiple-transactions-test.coffee [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+              {"error": "This is error :("}
+'''
+
+
+isApiElement = (element) ->
+  return false if element.element isnt 'category'
+  return element.meta?.classes.indexOf('api') isnt -1
+
+
+describe.only('Multiple Transactions', ->
```
Comment (kylef) [inline test/multiple-transactions-test.coffee:39]: @honzajavorek was the `.only` a mistake?

This disables all the other tests (which I might add are not passing in master).

Comment (honzajavorek) [reply test/multiple-transactions-test.coffee:39]: Yes, mistake 😱

**Pull request merged**